05week/spaceTravelToMars.js

```diff
@@ -1,51 +1,110 @@
-'use strict';
+"use strict";
 
-let assert = require('assert');
+let assert = require("assert");
 
 let jobTypes = {
-  pilot: 'MAV',
-  mechanic: 'Repair Ship',
-  commander: 'Main Ship',
-  programmer: 'Any Ship!'
+  pilot: "MAV",
+  mechanic: "Repair Ship",
+  commander: "Main Ship",
+  programmer: "Any Ship!",
 };
 
-// Your code here
+// class represents a crew memeber
+// crewmemeber should have a name, job, and specialSkill
+class CrewMember {
+  constructor(name, job, specialSkill) {
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+  }
+
+  //this method should ask THIS crewmember to the ship being passed in
+  //an entire ship instance is passed in, not just the name
+  //the entire crewmember is added to the ship's array of crew
+  enterShip(ship) {
+    ship.crew.push(this);
```

Comment:
you should set the local attribute `this.ship` to be the ship that is passed in.